Userland/Libraries/LibGL/GLContext.cpp

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2021, Jesse Buhagiar <jooster669@gmail.com>
+ * Copyright (c) 2021, Stephan Unverwerth <s.unverwerth@gmx.de>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "SoftwareGLContext.h"
+#include <LibGfx/Bitmap.h>
+
+__attribute__((visibility("hidden"))) GL::GLContext* g_gl_context;
```

Comment:
Is it possible to declare this static or use an anonymous namespace instead of using a gcc attribute?

Reply:
Static no, it's used in multiple files. But we could probably remove this attribute completely. I was just trying to prevent users of the library from directly mucking with the context variable, hence the `extern gl_context` in every file. But if anyone is extra willing to shoot themselves in the foot... meh.

Reply:
Oh, I didn't notice it was being used in multiple translation units. In that case, using the attribute is probably the best way.

Reply:
@ccapitalK Yeah, I wasn't shure if this is attribute is ok to use. It's used in two places, the kernel and LibELF. Kling or Linus should decide 😅